### PR TITLE
[strings] Local Telegram/Instagram links + Turkish translations

### PR DESCRIPTION
**Pull request by MetehanOzyurek**
* Localized Instagram and Telegram links on help page (see https://github.com/organicmaps/organicmaps/issues/2003)
* Added missing Turkish translations
* Fixed some Turkish translations

### CODE REVIEW

File: android/src/com/mapswithme/util/Constants.java [hunk 1]
```diff
@@ -23,8 +23,8 @@
     public static final String FB_OM_COMMUNITY_HTTP = "https://www.facebook.com/OrganicMaps";
     public static final String FB_OM_COMMUNITY_NATIVE = "fb://profile/102378968471811";
     public static final String TWITTER = "https://twitter.com/OrganicMapsApp";
-    public static final String TELEGRAM = "https://t.me/OrganicMaps";
-    public static final String INSTAGRAM = "https://www.instagram.com/organicmaps.app";
+    public static final String TELEGRAM = String TG_LINK;
```
Comment (rtsisyk) [inline android/src/com/mapswithme/util/Constants.java:26]: How does it work?

Comment (MetehanOzyurek) [reply android/src/com/mapswithme/util/Constants.java:26]: I don't know. I've just tried to make similar with other codes. So I'm not sure this works correctly.

**Comment by MetehanOzyurek** [comment]
@rtsisyk any update on this?

**Comment by MetehanOzyurek** [comment]
Also is it possible to add missing string translations in this update?

**Comment by dbf256** [comment]
> Also is it possible to add missing string translations in this update?

I think that it is not possible to merge part of pull request only, better to create a new one without java changes.

**Review by vng — APPROVED** [review]

**Pull request merged**